### ISSUE TITLE
In case of --no-lvm-storage confd raises error because it cannot find volume_group_name

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#435]
Originally reported of Google Code with ID 379.

```
<b>What steps will reproduce the problem?</b>
1. Init cluster with --no-lvm-storage
2. gnt-cluster info
3. workaround was to manually add "volume_group_name": "" in config.data cluster section.

<b>What is the expected output? What do you see instead?</b>
In conf-daemon.log:
2013-02-19 16:07:59,600321000000 EET: ganeti-confd pid=9984 ERROR Failed to load config: parsing configuration: key 'cluster': key 'volume_group_name' not found, object contains only....

<b>Please use labels and text to provide additional information.</b>
```
Originally added on 2013-02-19 14:15:47 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2013-02-20 14:53:40 +0000 UTC.

Added to Milestone: Release2.7

**Comment by Ganeti-Issues-Migrator** [comment]
```
Hi Dimistris, Iustin, and Helga (since you were cc-ed on this already),

We should probably get this fixed before beta2, hopefully very soon.
When is the earliest we could work on it? It seems to be the only critical missing piece before beta2 can be done.

Thanks,

Guido

```
Originally added on 2013-02-26 15:52:04 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
Issue 388 has been merged into this issue.
```
Originally added on 2013-02-27 10:56:18 +0000 UTC.

Mentioned Issue with GoogleCode ID 388 has been migrated to GitHub with ID #444.

**Comment by Ganeti-Issues-Migrator** [comment]
```
Hi,

If I recall correctly this was fixed: is that correct (and can be issue be marked as fixed, then).

Thanks,

Guido
```
Originally added on 2013-03-11 13:18:46 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2013-04-05 08:21:30 +0000 UTC.

Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
Marking 2.7 bugs fixed, as 2.7.0~rc1 was released, and should be ready to be tested in production environments.
```
Originally added on 2013-05-09 05:42:48 +0000 UTC.

Changed State: Released
